hack/validate/swagger

```diff
@@ -8,6 +8,9 @@ files=( $(validate_diff --diff-filter=ACMR --name-only -- 'api/swagger.yaml' ||
 unset IFS
 
 if [ ${#files[@]} -gt 0 ]; then
-	yamllint -c ${SCRIPTDIR}/.swagger-yamllint api/swagger.yaml
-	swagger validate api/swagger.yaml
+	(
+		export LANG=C.UTF-8
```

Comment:
Using a sub-shell so that I could export; not sure if that's needed (or just setting the variable is sufficient). Trouble is that I cannot reproduce the encoding problem locally (only in CI), and verified that exporting `LANG=C.UTF-8` fixes the issue

Reply:
Yeah, a subshell shouldn't be necessary -- just doing `LANG=C.UTF-8 yamllint ...` should have the same effect (unless `swagger` needs `LANG` set also).

Reply:
👍 updated